src/lemlib/chassis/differential.cpp

```diff
@@ -33,10 +33,10 @@ std::shared_ptr<pros::MotorGroup> makeMotorGroup(const std::initializer_list<int
  * A notable exception is the odometry, which at the moment is too complex to
  * construct in the initializer list
  */
-Differential::Differential(Drivetrain drivetrain, ControllerSettings linearSettings, ControllerSettings angularSettings,
-                           OdomSensors sensors)
+Differential::Differential(Drivetrain drivetrain, ControllerSettings<Length> lateralSettings,
+                           ControllerSettings<Angle> angularSettings, OdomSensors sensors)
     : drivetrain(drivetrain),
-      linearSettings(linearSettings),
+      lateralSettings(lateralSettings),
```

Comment:
```suggestion
      linearSettings(linearSettings),
```
change lateral to linear